src/applications/post-911-gib-status/actions/post-911-gib-status.js

```diff
@@ -43,24 +41,18 @@ export function getEnrollmentData() {
             // EVSS partner service has no record of this user
             return dispatch({ type: NO_CHAPTER33_RECORD_AVAILABLE });
           }
-          return Promise.reject(
-            new Error(
-              `post-911-gib-status getEnrollmentData() received unexpected error: ${
-                error.status
-              }: ${error.title}: ${error.detail}`,
-            ),
+          Sentry.captureException(
+            `post-911-gib-status getEnrollmentData() received unexpected error: ${
+              error.status
+            }: ${error.title}: ${error.detail}`,
           );
-        }
-        return Promise.reject(
-          new Error(
+        } else {
+          Sentry.captureException(
             'post-911-gib-status getEnrollmentData() received unexpected error (no status code available)',
-          ),
-        );
-      },
-    ).catch(error => {
-      Sentry.captureException(error);
-      return dispatch({ type: GET_ENROLLMENT_DATA_FAILURE });
-    });
+          );
+        }
+        return dispatch({ type: GET_ENROLLMENT_DATA_FAILURE });
```

Comment:
Removing the previous `catch` means that the call to `dispatch` can be moved into the new `catch`.  The appropriate `Sentry` calls will happen and then we call `dispatch` - just like before.